### ISSUE TITLE
Refresh data?

### ISSUE THREAD

**Issue by lxbarth** [osmlab/nycbuildings#138]
- Building data has been updated in December 2014 
https://data.cityofnewyork.us/Housing-Development/Building-Footprints/tb92-6tj8
- Address data has been updated in October 2014 https://data.cityofnewyork.us/City-Government/NYC-Address-Points/4iq4-tuhq

@colinreilly - does it makes sense to use these datasets to do an update to OSM or do you have an update coming up?

**Comment by colinreilly** [comment]
Currently, the data is published quarterly. We're looking to move towards automating it nightly/weekly. 

An update will be going up in the next week or two so it's probably best to wait for that. 

Ultimately it would be awesome to work out a way of automating the creation of tasks to implement the updates. @lxbarth Thoughts?

**Comment by lxbarth** [comment]
> Ultimately it would be awesome to work out a way of automating the creation of tasks to implement the updates. @lxbarth Thoughts?

Yeah, that'd be really cool. As a first, I'd like to do a manual compare and see how we stack up against current data in OSM. 

> An update will be going up in the next week or two so it's probably best to wait for that.

Will the format change in some way? If not, we can just use the current data for getting a head start.

**Comment by colinreilly** [comment]
It'll be the same format (shapefile). We could publish in a different format as well.

**Comment by lxbarth** [comment]
Shapefile is great.

**Comment by planemad** [comment]
The latest release is May 2016 https://data.cityofnewyork.us/Housing-Development/Building-Footprints/nqwf-w8eh

Conflating this with OSM every quarter looks to be an unsolved challenge. Did anyone make any progress on this?

**Comment by grischard** [comment]
Comparing old and new data would be useful to import a diff.

The makefile says:

```
NYC_AddressPoint.zip:
	curl -L "http://cl.ly/080a2K2A3y0k/download/NYC_AddressPoint.zip" -o NYC_AddressPoint.zip
```

but that link is long dead.

Would anyone in this  ticket still have a `NYC_AddressPoint.zip` on their drive?
